ansible/group_vars/all/vars.yml

```diff
@@ -35,3 +35,7 @@ nvm:
   directory: "/usr/local/bin/.nvm"
   script: "/usr/local/bin/.nvm/nvm.sh"
   version: "v0.40.0" # Derived from the git tag
+
+uv:
+  install_dir: /usr/local/bin
```

Comment:
I'm assuming you are not installing a binary but rather a folder with files and a binary. If so, it should be placed in `/use/local/lib`

Reply:
No, it only installs two binaries, therefore I thought `/usr/local/bin` was the most appropriate. But I can move it, if you find `/usr/local/lib` more appropriate

Reply:
Oh then lib is perfect!

Reply:
Whoops I meant bin :)